src/glauth.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+"""Provides glauth class to control glauth."""
```

Comment:
Just missing a newline between the module docstring and license header:

```suggestion
# See LICENSE file for licensing details.

"""Provides glauth class to control glauth."""
```

Reply:
Updated!